sphinx/screens-detail/join-tribe/join-tribe/src/main/java/chat/sphinx/join_tribe/ui/JoinTribeViewModel.kt

```diff
@@ -128,8 +128,8 @@ internal class JoinTribeViewModel @Inject constructor(
     }
 
     fun loadTribeData() {
-        if (tribeInfo != null) {
-            viewStateContainer.updateViewState(JoinTribeViewState.TribeLoaded(tribeInfo!!))
+        tribeInfo?.let { tribeInfo ->
```

Comment:
lambda variable name is the same as the global property and is "shadowed". we should shy away from this practice. I typically append the name with `nn` (not null) in this use case of `?.let` , so, `nnTribeInfo`